src/stores/attributes-returns.md

```diff
@@ -61,7 +61,13 @@ _Returns Attributes_
 
 1. In the left panel, choose **Manage Labels/Options**.
 
-1. Enter the **Default Store View**.
+1. In the **Manage Titles (Size, Color, etc.)** section, enter the title value for each Store View.
+
+{:.bs-callout-info}
+If the **Input Type** is `Dropdown` you can manage options in **Manage Options (Values of Your Attribute)** section.
+Click <span class="btn">Add Option</span> for adding new option, enter the title for Admin and each Store View.
+Click **Is Default** radiobutton, to make an option selected by default.
+Click **Delete** to delete an option.
```

Comment:
This would be better as a nested bullet list for the step.

```suggestion
1. In the **Manage Titles (Size, Color, etc.)** section, enter the label for each store view.

1. If the **Input Type** for the attribute is `Dropdown`, manage the options in the **Manage Options (Values of Your Attribute)** section.

   - To add an option, click <span class="btn">Add Option</span> and enter the label for Admin and each store view.
   - To make an option selected by default, choose **Is Default**.
   - To remove an option, click **Delete**.
```